org.godotengine.Godot.appdata.xml

```diff
@@ -1,16 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
   <id>org.godotengine.Godot.desktop</id>
-  <name>Godot</name>
-  <summary>Godot game engine editor</summary>
+  <name>Godot Engine</name>
+  <summary>Easily create 2D and 3D games</summary>
   <developer_name>The Godot Engine Community</developer_name>
   <description>
-    <p>
-      <em>NOTE: This Flatpak is unofficial and not supported by Godot Engine developers.</em>
-      Find officially supported binaries on godotengine.org.
-    </p>
-    <p>Godot is an advanced, feature-packed, multi-platform 2D and 3D open source game engine.</p>
-    <p>Create games with ease, using Godot's unique approach to game development.</p>
+    <p>The feature-packed, cross-platform game engine to create 2D and 3D games from a unified interface. Godot Engine provides a comprehensive set of common tools, so that you can focus on making games without having to reinvent the wheel. Games can be exported in one click to a number of platforms, including the major desktop platforms (Linux, macOS, Windows) as well as mobile (Android, iOS), web-based (HTML5) platforms and consoles (Switch, PS4 and Xbox One - via thirdparty publishers).</p>
+    <p>Completely free and open source under the very permissive MIT license. No strings attached, no royalties, nothing. Your game is yours down to the last line of engine code. Godot's development is fully independent and community-driven, empowering you to help shape the engine to match your expectations. It is supported by the Software Freedom Conservancy not-for-profit.</p>
```

Comment:
Godot is no longer under the Software Freedom Conservancy, but under its own foundation, the [Godot Foundation](https://godot.foundation).

Reply:
Ah, good catch; that was copy-pasted from the other app store listings. Those probably need to be updated. 😄 

```suggestion
    <p>Completely free and open source under the very permissive MIT license. No strings attached, no royalties, nothing. Your game is yours down to the last line of engine code. Godot's development is fully independent and community-driven, empowering you to help shape the engine to match your expectations. It is supported by the non-profit Godot Foundation.</p>
```